packages/esm-patient-orders-app/src/components/order-details-table.scss

```
@@ -30,19 +31,66 @@
   justify-content: flex-end;
 }
 
-.singleLineText {
+.priorityPill {
+  @include type.type-style('label-01');
+  background-color: colors.$gray-20;
+  border-radius: spacing.$spacing-05;
+  padding: spacing.$spacing-01;
+  text-align: center;
+
+  &[data-urgency='routine'] {
+    background-color: colors.$green-20;
+  }
+
+  &[data-urgency='urgent'] {
+    background-color: colors.$red-20;
+  }
+}
+
+.statusPill {
+  @include type.type-style('label-01');
+  background-color: colors.$gray-20;
+  border-radius: spacing.$spacing-05;
+  padding: spacing.$spacing-01;
+  text-align: center;
+
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+
+  &[data-status='received'] {
+    background-color: colors.$blue-20;
+  }
+
+  &[data-status='in progress'] {
+    background-color: colors.$cyan-20;
+  }
+
+  &[data-status='on-hold'] {
```

```suggestion
  &[data-status='on-hold'] {
```

Should this be `&[data-status='on hold'] {` instead?

---

Whoops!